### ISSUE TITLE
[Feature Request]: ESP32 WIFI OR ETHERNET

### ISSUE THREAD

**Issue by manuugonz** [josephdadams/TallyArbiter#773]
### What feature would you like to see added to TallyArbiter?

Hello!

I have seen this wonderful project. I have been using VTALLY for a long time, but it is really outdated and for some time now with the new VMIX updates it has been giving problems. Its software is much more advanced and functional. 

I wanted to ask you if your device would be compatible with an ESP32 with ethernet or wifi library. This for small sites is not a problem because with the WIFI, it works great. But for larger sites with more people we are looking for an ethernet system that can meet these needs. Do you think it is possible to have a code that can work with ESP32 devices with wifi and ethernet input? And depending on whether you have the ethernet connected or not, try the wifi? Or a button that allows you to switch from one system to another?  This would make an all-terrain device, capable of being used in simpler terrains as well as in much more complex and large terrains.

We have previously used Olimex boards (which even have POE), for other projects and we would love to be able to adapt your program to such a board.

Would it be possible to talk to you to see the possibilities of such a system and if you see it compatible?

**Comment by josephdadams** [comment]
Yes, there are many ESP32 implementations for Tally Arbiter. You can read more about them here: https://josephdadams.github.io/TallyArbiter/docs/usage/sections/listener-clients

You can use wired relays as well.
